src/content/doc-tutorials/using-github-actions.mdx

```diff
@@ -25,16 +25,17 @@ jobs:
     - name: Git checkout
       uses: actions/checkout@v4
     - name: Start SurrealDB
-      uses: surrealdb/setup-surreal@v1
+      uses: surrealdb/setup-surreal@v2
       with:
         surrealdb_version: latest
         surrealdb_port: 8000
         surrealdb_username: root
         surrealdb_password: root
-        surrealdb_auth: true
-        surrealdb_strict: true
+        surrealdb_auth: false
+        surrealdb_strict: false
```

Comment:
These values were copied from the other repo's documentation. Let me know if we don't want to display those here as well.

Reply:
Those values look good to me 👍